nvflare/app_opt/flower/applet.py

```diff
@@ -26,6 +26,25 @@
 from nvflare.security.logging import secure_format_exception
 
 
+def get_partition_id(fl_ctx: FLContext):
```

Comment:
@yanchengnv could you check if this logic also works in production? I can test POC mode later.

Reply:
Tested POC and it works fine.